ENCODE_release.py

```diff
@@ -411,6 +411,20 @@ def associated_with_hela_data(self, data_type, data):
                 return True
         return False
 
+    def has_audit(self, accession):
+        # Another GET request for page frame.
+        audit = encodedcc.get_ENCODE(accession,
+                                     self.connection,
+                                     'page').get('audit', {})
+        if (audit.get('ERROR') is not None
+                or audit.get('NOT_COMPLIANT') is not None):
+            details = [v[0]['category'] for k, v in audit.items()]
+            message = 'WARNING: AUDIT on object: {}. SKIPPING!'.format(details)
+            print(message)
```

Comment:
did you mean to leave the print statements?  There are a couple added in the PR

Reply:
Yes.